sdk/tables/Azure.Data.Tables/api/Azure.Data.Tables.netstandard2.0.cs

```diff
@@ -221,6 +244,13 @@ public partial class TableServiceStats
         public Azure.Data.Tables.Models.GeoReplication GeoReplication { get { throw null; } }
     }
 }
+namespace Azure.Data.Tables.Queryable
+{
+    public static partial class TableQueryableExtensions
+    {
```

Comment:
What is this method used for? I found only one usage in negative case. Can we remove it?

Reply:
See usage in `TableClientQueryableTests`

Reply:
Ugh, it makes for rather inconvenient usage pattern. Can we go with QueryAsync that takes a lambda instead?